src/main/resources/assets/crimson/models/block/potted_crimson_thorn.json

```diff
@@ -0,0 +1,6 @@
+{
+	"parent": "block/flower_pot_cross",
```

Comment:
again here

Reply:
Fixed in 9f64c8574f19b61a662f9ecfd49a927533642067.